addon/components/pikaday-input.js

```diff
@@ -32,7 +32,10 @@ export default Component.extend(PikadayMixin, {
   },
 
   onPikadayClose() {
-    if (this.get('pikaday').getDate() === null || isEmpty(this.$().val())) {
+    if (
+      this.get('pikaday').getDate() === null ||
+      isEmpty(this.get('element.value'))
```

Comment:
this.elment = dom element,  don't think we can access dom properties using ember getter.
`this.get('element').value`